### PR TITLE
Add Mona, Andrew, Adeola, Anastasia, Nonna to mentors

### PR DESCRIPTION
**Pull request by cathy-codes**
## Description
Added Mona, Andrew, Adeola, Anastasia, Nonna to mentors, and images for Sakirat, Agoro, Tvisha, and Sahana.
Issue: [#137](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/137)

## Change Type
- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Screenshots
Before
Screenshot 2024-05-01 133951

After
Screenshot 2024-05-01 134001
Screenshot 2024-05-01 134014
Screenshot 2024-05-01 134024
Screenshot 2024-05-01 134033
Screenshot 2024-05-01 134043
Screenshot 2024-05-01 134051

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -591,3 +591,148 @@
...
+  bio: |
+    I'm a Recruitment professional and start-up scaling specialist. I can help mentees with the soft-skills development to really make impact in their role and in the workplace.
+  image: assets/images/mentors/mona_ahluwalia.jpg
+  languages: English
+  skills:
+    experience: 5-7 Years
+    years: 7
+    mentee: I can help with all soft-skills related coaching
+    areas:
+      -
```
Comment (dricazenck) [inline _data/mentors.yml:612]: I guess it is missing areas and languages no?

File: _data/mentors.yml [hunk 2]
```diff
@@ -591,3 +591,148 @@
...
+    languages: 
+    focus:
+      - Switch from IC to management position
+    extra: Resume review, emotional intelligence, communication, interviewing
+  network:
+    - linkedin: https://www.linkedin.com/in/mona-ahluwalia-certrp-7754767a/
+
+- name: Andrew King 
+  disabled: false
+  sort: 100
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  sort: 10
```


Only ad-hoc active in the month will have sorting 100

File: _data/mentors.yml [hunk 3]
```diff
@@ -591,3 +591,148 @@
...
+  hours: 3
+  position:  Full stack Engineer
+  type: ad-hoc
+  index: 21
+  location: London, UK
+  bio: |
+    I am Frontend engineer with 4years of experience. I have had opportunities of working with several companies and building sustainable products for them. I am very keen to solving problems and helping people as well
+  image: assets/images/mentors/adeola_adekoyejo.jpg
+  languages: English
+  skills:
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
availability: [ 5 ]
skills:
```

**Review by dricazenck — COMMENTED** [review]
Please could you add for mentors that are ad-hoc or both the session availability?

File: _data/mentors.yml (hunk 1 above)
Comment (cathy-codes) [reply _data/mentors.yml:612]: Mona has no programming languages listed on the original website, and only one bullet point for areas which I've put into focus (I don't think it allowed me to put an empty focus).
Screenshot 2024-05-01 144550
Screenshot 2024-05-01 144557

**Review by dricazenck — APPROVED** [review]

**Review by dricazenck — APPROVED** [review]

**Pull request merged**